index.html

```diff
@@ -4892,11 +4873,11 @@ <h3>Base Context</h3>
         "credentialStatus": {"@id": "cred:credentialStatus", "@type": "@id"},
         "credentialSubject": {"@id": "cred:credentialSubject", "@type": "@id"},
         "evidence": {"@id": "cred:evidence", "@type": "@id"},
-        "expirationDate": {"@id": "cred:expirationDate", "@type": "xsd:dateTime"},
+        "validUntil": {"@id": "cred:validUntil", "@type": "xsd:dateTime"},
         "holder": {"@id": "cred:holder", "@type": "@id"},
         "issued": {"@id": "cred:issued", "@type": "xsd:dateTime"},
         "issuer": {"@id": "cred:issuer", "@type": "@id"},
-        "issuanceDate": {"@id": "cred:issuanceDate", "@type": "xsd:dateTime"},
+        "validFrom": {"@id": "cred:validFrom", "@type": "xsd:dateTime"},
```

Comment:
Note that `validUntil` and `validFrom` are already present in the context, at the end -- so these are duplicates.

Reply:
good catch, should be resolved now.